app/pages/solutions-list/filterType/covid19/ui.test.js

```diff
@@ -0,0 +1,105 @@
+import nock from 'nock';
+import { Selector, ClientFunction } from 'testcafe';
+import aFoundationSolutionList from '../../../../test-utils/fixtures/aFoundationSolutionList.json';
+import { extractInnerText } from '../../../../test-utils/helper';
+
+const mocks = async (responseStatus, responseBody) => {
+  await nock('http://localhost:5100')
+    .get('/api/v1/Solutions/Foundation')
+    .reply(responseStatus, responseBody);
+};
+
+const pageSetup = async (
+  t, responseStatus = 200, responseBody = aFoundationSolutionList,
+) => {
+  await mocks(responseStatus, responseBody);
+  await t.navigateTo('http://localhost:1234/solutions/foundation');
+};
+
+fixture('Show Foundation Solution List Page')
```

Comment:
change title

Reply:
need to change this to work with the covid19 page - currently it is the same as the foundation page